android/app/build.gradle

```diff
@@ -55,6 +65,12 @@ android {
         pickFirst "lib/x86_64/libc++_shared.so"
         pickFirst "lib/x86/libc++_shared.so"
```

Comment:
You can add 

     buildConfigField 'boolean', 'USE_FLIPPER', project.ext.react.enableFlipper

inside `defaultConfig`. Then you'll be able to use `BuildConfig.USE_FLIPPER` in `TestAppReactNativeHost.kt` instead of `BuildConfig.DEBUG`, and then you can remove the try/catch block :)

Reply:
Then we also match the logic on iOS

Reply:
`enableFlipper` could be turned on but it won't be included in release builds. That's why we use reflection here. Is there a way to have `BuildConfig.USE_FLIPPER` only be true in debug builds?

Reply:
Yes, okay, so instead of using `defaultConfig` you can add this below:

    android {
       ...
      buildTypes {
        release {
            buildConfigField 'boolean', 'USE_FLIPPER', false
        }

        debug {
            buildConfigField 'boolean', 'USE_FLIPPER', project.ext.react.enableFlipper
        }
      }
      ..
    }

Reply:
Hmm... But we still need to `import` on top, right?

Reply:
Talked offline. Think what you have is fine